lib/features/profiles/models/donation.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:equatable/equatable.dart';
```

Comment:
I would suggest to move it to the models folder of the feature

Reply:
Alright but it is shared with the wallet screen//profile model.